client/state/activity-log/log/reducer.js

```diff
@@ -1,18 +1,34 @@
 /** @format */
 /**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+
+/**
  * Internal dependencies
  */
-import { logItemsSchema } from './schema';
+import ActivityQueryManager from 'lib/query-manager/activity';
 import { ACTIVITY_LOG_UPDATE } from 'state/action-types';
 import { createReducer, keyedReducer } from 'state/utils';
 
+// FIXME: Restore persistence after implementing queryManager
+//import { logItemsSchema } from './schema';
+
+const debug = debugFactory( 'calypso:state:activity-log:reducer' );
+
 // FIXME: No-op reducers
 export const logError = keyedReducer( 'siteId', state => state );
 
 export const logItems = keyedReducer(
```

Comment:
Found it 🤦‍♂️ e6c622011a149f3f405c0a664c90e642592b1c66

The constructor and `receive` methods have slightly different signatures.